src/input/__test__/Input_test.jsx

```diff
@@ -65,4 +65,16 @@ describe('Input test', () => {
     expect(w.find('.el-textarea__inner').first().prop('style').resize).toBe('both');
   });
 
+  it('onChange', (done) => {
+    const updateValue = function(event) {
+      expect(event.target).toBeInstanceOf(EventTarget);
```

Comment:
I know that this is not perfect, but as SyntheticEvent is not a constructor, just a type, and that I don't really get an Event, I needed a way to make sure I was getting a SyntheticEvent rather than a string. Events have a target, so I didn't want to check on every single property. Any suggestion is welcome if you have a better idea.

Reply:
`dInputelay `?

Reply:
Personally, I think it would be better that passing the event than the string. However this will change the API and other users will need to update their old code.

Reply:
Sorry for the lazy response, passing the value string is an easy way to devs, I think we can pass additional parameter as `SyntheticEvent` type.